### DISCUSSION TITLE
What causes useFormContext becomes null

### DISCUSSION THREAD

**Discussion by Hud97** [react-hook-form/react-hook-form#9238] [Q&A]
Hey communities, I recently encountered an error saying `useFormContext `becomes null, I have this panel that has tabs and each tab has their own form, I wrapped this panel with `FormProvider `and everything works well but as soon as I close the panel, it throws the error.

I dont have sandbox to reproduce it, but it would be nice to know what causes useFormContext to becomes null, thanks :)

**Comment by Moshyfawn** [comment]
There shouldn't be anything in RHF that would cause the context value to become `null`.

Please, try to provide a minimal reproduction codesandbox with the behaviour you're experiencing
